src/sqlancer/doris/gen/DorisExpressionGenerator.java

```diff
@@ -0,0 +1,461 @@
+            case BOOLEAN:
+                if (!globalState.getDbmsSpecificOptions().testBooleanConstants) {
+                    throw new IgnoreMeException();
+                }
+                return DorisConstant.createBooleanConstant(Randomly.getBoolean());
+            default:
+                System.out.println("ERROR TYPE:" + columnDataType);
+                throw new AssertionError();
+        }
+    }
+
+    @Override
+    public List<Node<DorisExpression>> generateOrderBys() {
+        List<Node<DorisExpression>> expr = super.generateOrderBys();
+        List<Node<DorisExpression>> newExpr = new ArrayList<>(expr.size());
+        for (Node<DorisExpression> curExpr : expr) {
+            if (Randomly.getBoolean()) {
+                curExpr = new NewOrderingTerm<>(curExpr, Ordering.getRandom());
+            }
+            newExpr.add(curExpr);
+        }
+        return newExpr;
+    };
+
+    public static class DorisCastOperation extends NewUnaryPostfixOperatorNode<DorisExpression> {
+
+        public DorisCastOperation(Node<DorisExpression> expr, DorisCompositeDataType type) {
+            super(expr, new Operator() {
+
+                @Override
+                public String getTextRepresentation() {
+                    return "::" + type.toString();
+                }
+            });
+        }
+
+    }
+
+    public enum DorisAggregateFunction {
+        MAX(1), MIN(1), AVG(1), COUNT(1), STRING_AGG(1), FIRST(1), SUM(1), STDDEV_SAMP(1), STDDEV_POP(1), VAR_POP(1),
+        VAR_SAMP(1), COVAR_POP(1), COVAR_SAMP(1);
+
+        private int nrArgs;
+
+        DorisAggregateFunction(int nrArgs) {
+            this.nrArgs = nrArgs;
+        }
+
+        public static DorisAggregateFunction getRandom() {
+            return Randomly.fromOptions(values());
+        }
+
+        public int getNrArgs() {
+            return nrArgs;
+        }
+
+    }
+
+    public enum DBFunction {
+        // trigonometric functions
+        ACOS(1), //
+        ASIN(1), //
+        ATAN(1), //
+        COS(1), //
+        SIN(1), //
+        TAN(1), //
+        COT(1), //
+        ATAN2(1), //
+        // math functions
+        ABS(1), //
+        CEIL(1), //
+        CEILING(1), //
+        FLOOR(1), //
+        LOG(1), //
+        LOG10(1), LOG2(1), //
+        LN(1), //
+        PI(0), //
+        SQRT(1), //
+        POWER(1), //
+        CBRT(1), //
+        ROUND(2), //
+        SIGN(1), //
+        DEGREES(1), //
+        RADIANS(1), //
+        MOD(2), //
+        XOR(2), //
+        // string functions
+        LENGTH(1), //
+        LOWER(1), //
+        UPPER(1), //
+        SUBSTRING(3), //
+        REVERSE(1), //
+        CONCAT(1, true), //
+        CONCAT_WS(1, true), CONTAINS(2), //
+        PREFIX(2), //
+        SUFFIX(2), //
+        INSTR(2), //
+        PRINTF(1, true), //
+        REGEXP_MATCHES(2), //
+        REGEXP_REPLACE(3), //
+        STRIP_ACCENTS(1), //
```

Comment:
It seems these were copied from DuckDB? I couldn't find, for example, `STRIP_ACCENTS` in the Apache Doris documentation (see https://doris.apache.org/docs/sql-manual/sql-functions/string-functions/).